.travis.yml

```diff
@@ -1,21 +1,23 @@
 language: python
 python:
-- '2.7'
-- '3.3'
-- '3.4'
-- '3.5'
-- '3.6'
+  - '2.7'
+  - '3.4'
```

Comment:
I dropped Python 3.3 as [this version of Python has reached end-of-life](https://www.python.org/downloads/release/python-337/).